### ISSUE TITLE
will lr_scheduler.FactorScheduler bring different loss?

### ISSUE THREAD

**Issue by umichyiwan** [apache/incubator-mxnet#3550]
I am fine-tuning a model using

```
model_args['lr_scheduler'] = mx.lr_scheduler.FactorScheduler(
            step=max(1, int(self.epoch_size * self.lr_factor_epoch)),
            factor=self.lr_factor)
model = mx.model.FeedForward(
            ctx=self.ctx,
            symbol=net,
            num_epoch=self.num_epoch,
            learning_rate=self.lr,
            momentum=self.momentum,
            wd=self.weight_decay,
            initializer=mx.init.Xavier(factor_type="in", magnitude=2.34),
            begin_epoch=begin_epoch,
            # **model_args
)
```

I found it very strange that , for first several batches, using model_args and not using it brings me two different loss (same training data), 0.01 and 0.09 respectively.
Here self.epoch_size = 40000, self.lr_factor_epoch = 2, self.lr_factor = 0.1.
Anyone could explain why this happens?

**Comment by yajiedesign** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
